docs/what-is-rocm.rst

```diff
@@ -10,6 +10,9 @@ ROCm is an open-source stack, composed primarily of open-source software, design
 graphics processing unit (GPU) computation. ROCm consists of a collection of drivers, development
 tools, and APIs that enable GPU programming from low-level kernel to end-user applications.
 
+.. figure:: data/rocm-sw-stack-chart.png
```

Comment:
I see the filename is causing spelling errors.  Perhaps we:
1) Use the JPG
2) filename change to rocm-software-stack-6_1_0.jpg

Reply:
"jpg" is also flagged by the linter 😭

Reply:
Fixed